catalog/ui/search-ui-app/pom.xml

```diff
@@ -17,8 +17,8 @@
     xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>ddf.ui.search</groupId>
-        <artifactId>search-ui</artifactId>
+        <groupId>ddf.ui</groupId>
+        <artifactId>ui</artifactId>
```

Comment:
Sorry i was going through this too quickly and didn't see that this was for the parent.